microprofile/cdi/src/main/java/io/helidon/microprofile/cdi/HelidonContainerImpl.java

```diff
@@ -119,7 +122,7 @@ final class HelidonContainerImpl extends Weld implements HelidonContainer {
 
     HelidonContainerImpl() {
         this.bootstrap = new WeldBootstrap();
-        id = UUID.randomUUID().toString();
+        this.id = STATIC_INSTANCE;
```

Comment:
What was this id used for? Moving from a UUID to a static value is a big change, not sure about the implications.

Reply:
I checked it with @tomas-langer because I was also thinking it could have implications

There is only one per JVM, and he was fine to give that value of "STATIC_INSTANCE" if will make TCKs to pass